### PR TITLE
Fix related item review state 376

### PR DESCRIPTION
**Pull request by arsenico13**
Fix the WorkflowException when requesting a fullobject with a related item that doesn't have a workflow (see issue #376).

I added one test about this topic. When tryint to run all `TestContentGet` I saw that all of them were skipped. I had to remove the check at the top of the test class:
```
if not HAS_AT:
    raise unittest.SkipTest("Skip tests if Archetypes is not present")
```

because this is a DX test.

For compatibilty reasons, do we need to add a HAS_DX check here?

### CODE REVIEW

**Comment by arsenico13** [comment]
@jenkins-plone-org please run jobs

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -2,17 +2,20 @@
     place names and roles of the people who contribute to this package
     in this file, one to a line, like so:
 
-- Timo Stollenwerk, Original Author
```
Comment (tisto) [inline CONTRIBUTORS.rst]: @arsenico13 we usually just append names at the end as a best practice in the Plone community. I appreciate that you want to sort the authors by name but this never really worked. Most people add their names at the end, so I think we should keep things simple and just always append the names at the end...

File: src/plone/restapi/tests/test_content_get.py [hunk 2]
```diff
@@ -23,8 +25,6 @@ class TestContentGet(unittest.TestCase):
     layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
 
     def setUp(self):
-        if not HAS_AT:
```
Comment (tisto) [inline src/plone/restapi/tests/test_content_get.py]: @arsenico13 does this test pass with AT?

File: CONTRIBUTORS.rst (hunk 1 above)
Comment (arsenico13) [reply CONTRIBUTORS.rst]: Sorry, didn't know that!
Restored the old order and added missing contributors!

File: src/plone/restapi/tests/test_content_get.py (hunk 2 above)
Comment (arsenico13) [reply src/plone/restapi/tests/test_content_get.py]: Here the test is using the dexterity layer. The ones with AT are at the bottom of the file, line 181.

**Comment by arsenico13** [comment]
Hi @tisto, about your question:
> @arsenico13 does this test pass with AT?
Here the tests are using the dexterity layer. I removed those lines because they were actually preventig those tests from executing (the ones with AT are at the bottom of the file, line 181)
but now I added a new check and if dexterity is not present, the tests are skipped.
:)

**Comment by arsenico13** [comment]
I have another question.
All the tests run successfully but [here](https://jenkins.plone.org/job/pull-request-5.2/838/console) there is an error message in the console:

> ERROR: Step ‘Publish JUnit test result report’ failed: No test report files were found. Configuration error?

and I don't know what it is, sorry :(

**Comment by arsenico13** [comment]
@jenkins-plone-org please run jobs

**Comment by tisto** [comment]
@arsenico13 this seems to be a Jenkins issue. We have troubles with Jenkins so I'd be ok to merge if Travis passes.

**Comment by tisto** [comment]
@arsenico13 let's see if we can get a green Jenkins. If not I will merge and release over the weekend.

**Pull request merged**